labs/oas-apigee-mock/lab.md

```diff
@@ -0,0 +1,181 @@
+id: oas-apigee-mock
+
+# OAS Apigee Mock Lab
```

Comment:
I am thinking maybe change the theme to TDD or BDD proxy development instead?

The mocking part is one aspect but IMO you're telling a bigger story here.